html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li>
+      <a href="../index.html">Home Page</a>
+    </li>
+    <li>
+      <a href="../about-me/about-me.html">About Me</a>
```

Comment:
On this line, you are exiting the current directory just to go straight back into it. Instead, you should just state the name of the file directly.